sandbox/extension/distribution_2021-02-15.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<?xml-stylesheet type='text/xsl' href='/style/human.xsl'?>
+<extension xmlns='http://rs.gbif.org/extension/'
+    xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
+    xmlns:dc='http://purl.org/dc/terms/'
+    xsi:schemaLocation='http://rs.gbif.org/extension/ https://raw.githubusercontent.com/gbif/rs.gbif.org/master/schema/extension_2021-02-15.xsd'
+    rowType='http://rs.gbif.org/terms/1.0/Distribution'
+    dc:title='Species Distribution'
+    name='Distribution'
+    namespace='http://rs.gbif.org/terms/1.0'
+    dc:issued='2021-02-15'
+    dc:subject='dwc:Taxon'
+    dc:relation='http://rs.gbif.org/extension/gbif/1.0/distribution.xml'
+    dc:description='Geographic distribution of a taxon. Replaces version issued 2020-07-15 to match latest release of the Darwin Core standard.'>
+
+    <property name='locationID' namespace='http://rs.tdwg.org/dwc/terms/' qualName='http://rs.tdwg.org/dwc/terms/locationID' dc:relation='https://dwc.tdwg.org/terms/#locationID' dc:description='An identifier for the set of location information (data associated with dcterms:Location). May be a global unique identifier or an identifier specific to the data set.' examples='&quot;https://opencontext.org/subjects/768A875F-E205-4D0B-DE55-BAB7598D0FD1&quot;' required='false'/>
```

Comment:
This is a drastic change to the original proposal which clearly recommends a few formats based on the prefix:id format.

Reply:
> The meaning of locationID isn't different, so that is good, but the usage notes (comments, which the new extension schema allows to be separate) should be augmented rather than replacing the definition of the term, which is normative.

I agree, but then lets leave those usage notes in. They are removed here.
We did want to recommend a specific use for identifying larger areas in particular, based on different existing systems.

> Similarly with countryCode, except that here two different vocabularies are being recommended at the same time. It is trivial and loss-less to convert 3-letter codes to two letter codes. Why change things?

Why change things is my question here really. This has been in use for over a decade.
It is the same ISO3166 standard in the end and neither conflicting nor ambiguous to use 2 or 3 letter codes. The same is true for languages. Btw, why is the GBIF distribution extension the only one being updated here? What about [vernacular names, descriptions, species profile, etc](https://rs.gbif.org/extension/gbif/1.0/)? They all have at least one dwc term. Species profile for example contains dwc:habitat which has a definition in DwC that does not make much sense for a species profile without any "event": 

> A category or description of the habitat in which the Event occurred.

If we want to be strict with the correct usage of DwC terms it seems to me then that it is best to never use DwC terms outside their original context. And that DwC must also be very conservative and not easily change any definition at all. In general the application of dwc terms to species data has probably been a wrong decision in that light. But it had worked quite well so far :)

Reply:
> I see the schema location has been changed from gbif.org to github, e.g. https://raw.githubusercontent.com/gbif/rs.gbif.org/master/schema/extension_2021-02-15.xsd
> 
> Is this a good idea? github might change the way they offer access to "raw" files

Correct. It should not be an external URL. 

(OT: I anticipate in a future version of the GBIF schema repository that we explore using `git` as a way to have a distributed schema repo, to remove reliance on a central service at runtime similar to how wikimedia [manage](https://techblog.wikimedia.org/2020/09/17/wikimedias-event-data-platform-json-event-schemas/) their schemas.

Reply:
> This is a drastic change to the original proposal which clearly recommends a few formats based on the prefix:id format.

Is it a change at all to the definition? I don't think so. Since the usage notes are separated out into a Comments attribute now, the recommendation for what to do with locationID for this context should go there, as it isn't a normative part of the definition.

Reply:
> > I see the schema location has been changed from gbif.org to github, e.g. https://raw.githubusercontent.com/gbif/rs.gbif.org/master/schema/extension_2021-02-15.xsd
> > Is this a good idea? github might change the way they offer access to "raw" files
> 
> Correct. It should not be a GitHub URL.
> 
> (OT: I anticipate in a future version of the GBIF schema repository that we explore using `git` as a way to have a distributed schema repo, to remove reliance on a central service at runtime similar to how wikimedia [manage](https://techblog.wikimedia.org/2020/09/17/wikimedias-event-data-platform-json-event-schemas/) their schemas.

I used to GitHub file location not being able to divine in advance where you would end up putting it during or after testing.  I fully expected it to be changed, but I wanted to make sure it was known which version of the schema the XML was supposed to validate against.

Reply:
> Is it a change at all to the definition? I don't think so. Since the usage notes are separated out into a Comments attribute now, the recommendation for what to do with locationID for this context should go there, as it isn't a normative part of the definition.

No not a change. It is more specific actually and the given example contradicts that. But my main problem is that I cannot see any Comments attribute. I guess this is something in DwC which does not exist in the "IPT" definitions? I totally agree the separation makes most sense, but we then need to have those comments somewhere available. It is an application profile that tries to recommend more controlled values than "generic" DwC does. Even though this is from a long time ago it is in use by various parties and I still think it makes sense to define a DwC term in the application more rigid than in DwC per se.

Reply:
> > Is it a change at all to the definition? I don't think so. Since the usage notes are separated out into a Comments attribute now, the recommendation for what to do with locationID for this context should go there, as it isn't a normative part of the definition.
> 
> No not a change. It is more specific actually and the given example contradicts that. My main problem is that I cannot see any Comments attribute. We just need to revive those. I fully agree the separation makes most sense. It is an application profile that tries to recommend more controlled values than "generic" DwC does.

Understood. The locationID in Darwin Core has no Comments. So none appeared in the extension because I overwrote it with the Darwin Core. So yes, they need to be revived. It's also fine to make examples in the Extension definition that match the constrained recommendations and abandon the more generic ones from the Darwin Core.